documentation/modules/post/windows/manage/vss.md

```diff
@@ -0,0 +1,178 @@
+## Overview
+This module will perform management actions for Volume Shadow Copies on the system. This is based on the VSSOwn Script
+originally posted by Tim Tomes and Mark Baggett. The session must be running with Administrative privileges and outside
+of UAC.
+
+## Options
+
+### DEVICE
+
+*Only applicable with the VSS_MOUNT action.*
+
+DeviceObject of the shadow copy to mount. This should begin with `\\?\GLOBALROOT\Device` and **must end with a slash (`\`)**.
+
+### PATH
+
+*Only applicable with the VSS_MOUNT and VSS_UNMOUNT actions.*
+
+Path to use for mounting the shadow copy.
+
+### SIZE
+
+*Only applicable with the VSS_SET_MAX_STORAGE_SIZE action.*
+
+Size in bytes to set for max storage.
+
+### VOLUME
+
+*Only applicable with the VSS_CREATE action.*
+
+Volume to make a copy of.
+
+## Scenarios
+
+### Create And Access A Shadow Copy
+
+First, ensure the session is running with elevated privileges and that UAC is not restricting it.
+
+```
+msf6 post(windows/manage/vss) > 
+[*] Sending stage (200262 bytes) to 192.168.159.30
+[*] Meterpreter session 2 opened (192.168.159.128:4444 -> 192.168.159.30:62600) at 2021-01-04 12:09:59 -0500
+
+msf6 post(windows/manage/vss) > sessions -i -1
```

Comment:
So `-2` actually won't work, `-1` is a special value that means the last session that was opened. In this case, that's session 2 and `-1` allows me to refer to it generically. If and when session 3 is opened, `-1` will then refer to that one since it will be the last open one.

Reply:
Oh! my bad, I'm sorry. I've learned something today.

Reply:
https://blog.rapid7.com/2020/12/23/metasploit-tips-and-tricks-for-haxmas-2020-2/#quicklyinteractingwithnewsessions

🎉